.github/workflows/snyk.yml

```diff
@@ -0,0 +1,20 @@
+name: Snyk
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  security:
+    uses: guardian/.github/.github/workflows/sbt-node-snyk.yml@main
+    with:
+      DEBUG: true
+      ORG: guardian-dotcom-n2y
+      SKIP_NODE: true
+      
+      
+      JAVA_VERSION: 8
+    secrets:
+       SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
Bit of odd whitespace here…

How was Java 8 chosen?

Reply:
Whitespace should be fixed now. I think as there is no java version specified here, this is specified as a sensible default (it may also be Jon tested this and found it to work).